be/src/exec/pipeline/aggregate/spillable_aggregate_distinct_blocking_operator.cpp

```diff
@@ -29,6 +29,7 @@ bool SpillableAggregateDistinctBlockingSinkOperator::is_finished() const {
 }
 
 Status SpillableAggregateDistinctBlockingSinkOperator::set_finishing(RuntimeState* state) {
```

Comment:
should it check `if (_spill_strategy == spill::SpillStrategy::NO_SPILL) {` like above?